src/utils/helpers.js

```diff
@@ -194,7 +194,7 @@ const helpers = {
    * @inner
    */
   getFullExplorerURL(urlPath) {
-    return path.join(this.getExplorerURL(), urlPath);
+    path.join(this.getExplorerURL(), urlPath);
```

Comment:
chore: It's necessary to return this string, right?

Reply:
Definitely, fixed!

Thanks